### PR TITLE
Added `You own EVM RPC` section into tooling

### PR DESCRIPTION
**Pull request by akru**
**Description**

Short section about running own EVM RPC. 
Changes from https://github.com/AstarNetwork/Astar/pull/833 adopted.

### CODE REVIEW

File: docs/EVM/developer-tooling.md [hunk 1]
```diff
@@ -129,3 +129,19 @@ module.exports = {
 ```
 
 Deploy/Migrate by running `truffle migrate --network shibuya`, replacing `shibuya` with your chosen network. If `--network` is not specified, the network values under`development` will be used.
+
+## Your own RPC server
+
+For EVM developers and projects could be reasonable to have own EVM endpoints. In case of using public endpoints the project will depend of this endpoint, centralized EVM RPC can become single point of failure. Officially Astar team recommend projects to use their own managed EVM endpoints.
```
Comment (Dinonard) [inline docs/EVM/developer-tooling.md:135]:
```suggestion
For EVM developers and projects, it is reasonable to have their own managed EVM endpoints. Relying on public endpoints introduces, introduces the risk of relying on centralized endpoints which can become single point of failure.

Astar team officially recommends projects to use their own managed EVM endpoints.
```

File: docs/EVM/developer-tooling.md [hunk 2]
```diff
@@ -129,3 +129,19 @@ module.exports = {
...
+
+Fortunately launch Astar Network endpoint is not that difficult.
```
Comment (Dinonard) [inline docs/EVM/developer-tooling.md:137]:
```suggestion
Fortunately, launching Astar Network endpoint is not that difficult.
```

File: docs/EVM/developer-tooling.md [hunk 3]
```diff
@@ -129,3 +129,19 @@ module.exports = {
...
+
+> By default EVM RPC server is disabled, for turn it on please add `--enable-evm-rpc` flag into launch string.
```
Comment (Dinonard) [inline docs/EVM/developer-tooling.md:139]:
```suggestion
:::note
By default EVM RPC server is disabled, to turn it on please add `--enable-evm-rpc` flag into the launch string.
:::
```

I think you should mention this in the chapter which describes operating a node in general.

File: docs/EVM/developer-tooling.md [hunk 4]
```diff
@@ -129,3 +129,19 @@ module.exports = {
...
+
+```
+astar-collator --chain=shiden --enable-evm-rpc --unsafe-rpc-external --unsafe-ws-external
+```
+
+Launch string above start Astar Collator for Shiden network. HTTP endpoint becomes available on port `9933` and WS on port `9944`.
```
Comment (Dinonard) [inline docs/EVM/developer-tooling.md:145]:
```suggestion
Launch string above will start Astar Collator for Shiden network. HTTP endpoint becomes available on port `9933` and WS on port `9944`.
```

File: docs/EVM/developer-tooling.md [hunk 5]
```diff
@@ -129,3 +129,19 @@ module.exports = {
...
+
+We also could recommend take attention to `--ws-max-connections` parameter, by default it relatively small and you probably would increase it to couple of thousands.
```
Comment (Dinonard) [inline docs/EVM/developer-tooling.md:147]:
```suggestion
We also recommend to pay attention to the `--ws-max-connections` parameter. By default it relatively small and you will probably want to increase it to couple of thousands.
```

**Review by Dinonard — CHANGES_REQUESTED** [review]
Just some grammar improvements (Maarten will probably override me though :grin:).

And I recommend using `::note` to emphasize the fact EVM rpc is disabled by default.

**Comment by akru** [comment]
Thank you @Dinonard

**Pull request merged**